app/src/main/java/io/homeassistant/companion/android/sensors/NetworkSensorManager.kt

```diff
@@ -174,7 +176,25 @@ class NetworkSensorManager : SensorManager {
             conInfo = wifiManager.connectionInfo
         }
 
-        val bssid = if (conInfo!!.bssid == null) "<not connected>" else conInfo.bssid
+        var bssid = if (conInfo!!.bssid == null) "<not connected>" else conInfo.bssid
+
+        val settingName = "replace_$bssid"
+        val sensorDao = AppDatabase.getInstance(context).sensorDao()
+        val fullSensor = sensorDao.getSettings(bssidState.id)
+        val getNextBSSID = fullSensor?.settings?.firstOrNull { it.name == "get_next_bssid" }?.value ?: "false"
+        val currentSetting = fullSensor?.settings?.firstOrNull { it.name == settingName }?.value ?: ""
+        if (getNextBSSID == "true") {
+            if (currentSetting == "") {
+                sensorDao.add(Setting(bssidState.id, "get_next_bssid", "false", "toggle"))
+                sensorDao.add(Setting(bssidState.id, settingName, "", "string"))
+            }
+        } else {
+            if (currentSetting != "")
+                bssid = currentSetting
```

Comment:
Maybe if `currentSetting == ""` we should delete the setting?

Reply:
Well if the user did not change the setting fast enough it would remove it so I don't think that would be friendly.  The page where they create the entries updates every 10 seconds so I wanted to be mindful of that :)

Reply:
Above I suggest that we change the default to the string of the BSSID rather than `""`